radio/src/datastructs_private.h

```diff
@@ -889,7 +889,7 @@ PACK(struct RadioData {
   NOBACKUP(int8_t   pwrOnSpeed:3);
   NOBACKUP(int8_t   pwrOffSpeed:3);
   NOBACKUP(uint8_t  imperial:1);
-  NOBACKUP(uint8_t  jitterFilter:1); /* 0 - active */
+  NOBACKUP(uint8_t  noJitterFilter:1); /* 0 - active */
```

Comment:
Since you caught the others... `0 - Jitter filter active`

Reply:
I have added a commit to the branch that changes the text in the comment.